examples/examples/homogeneous_elastic/forward_2D.py

```diff
@@ -0,0 +1,122 @@
+
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.lines as lines
+
+from stride import *
+from stride.utils import wavelets
+
+
+async def main(runtime):
+    # Create the grid
+    extent = (120e-3, 70e-3)  # [m]
+    spacing = (0.5e-3, 0.5e-3)  # [m]
+
+    shape = tuple([int(each_extent / each_spacing) + 1 for each_extent, each_spacing in zip(extent, spacing)])  # [grid_points]
+    extra = (50, 50)  # boundary [grid_points]
+    absorbing = (40, 40)  # absorbing boundary [grid_points]
+
+    space = Space(shape=shape,
+                  extra=extra,
+                  absorbing=absorbing,
+                  spacing=spacing)
+    print('Space complete')
+
+    start = 0.  # [s], start = 0.
+    step = 7.855e-8  # [s], step = 0.08e-6
+    end = 1.e-04/2  # [s]
+
+    time = Time(start=start,
+                step=step,
+                stop=end)
+    print('Time complete')
+
+    # Create problem
+    problem = Problem(name='test2D_elastic',
+                      space=space, time=time)
+    print('Problem complete')
+
+    # Create medium
+    vp = ScalarField(name='vp', grid=problem.grid)
+    vp.fill(1500.)  # [m / s]
+
+    vs = ScalarField(name='vs', grid=problem.grid)
+    vs.fill(1000.)  # [m / s]
+
+    rho = ScalarField(name='rho', grid=problem.grid)
+    rho.fill(1000.)  # [g / cm^3]
```

Comment:
this should be [kg / m^3], which is the SI unit

Reply:
Correction made